scheds/rust/scx_layered/src/main.rs

```diff
@@ -1486,15 +1497,21 @@ impl<'a, 'b> Scheduler<'a, 'b> {
                     preempt_first,
                     exclusive,
                     nodes,
+                    slice_us,
                     ..
                 } => {
+                    layer.slice_ns = if *slice_us > 0 {
+                        *slice_us * 1000
+                    } else {
+                        opts.slice_us
+                    };
                     layer.min_exec_ns = min_exec_us * 1000;
                     layer.yield_step_ns = if *yield_ignore > 0.999 {
                         0
                     } else if *yield_ignore < 0.001 {
-                        opts.slice_us * 1000
+                        layer.slice_ns * 1000
```

Comment:
I guess this conversion doesn't need to be done since it already gets converted above...